src/include/daos/tests_lib.h

```diff
@@ -337,5 +337,18 @@ int dmg_storage_device_list(const char *dmg_config_file, int *ndisks,
  */
 int dmg_storage_set_nvme_fault(const char *dmg_config_file,
 			       char *host, const uuid_t uuid, int force);
-
+/**
+ * Get NVMe Device health stat.
+ *
+ * \param dmg_config_file
+ *		[IN]	DMG config file
+ * \param host	[IN]	Nvme set to faulty on host name provided. Only single
+					disk can be set to fault for now.
+ * \param uuid	[IN]	UUID of the device.
+ * \param stat	[IN/OUT]
```

Comment:
Slightly confusing...
(sp) stats
[in] Health stat name to query
[out] Current health stat

Reply:
Fixed (SP).
This function will to query the Health stats for example Write Errors:0, INPUT will be "Write Errors" and out is the counter "0"

Reply:
right, I was saying the descriptions of the IN/OUT params were confusing

Reply:
Ah got it let me change the description